tests/entrypoints/openai/test_chat_utils.py

```diff
@@ -33,8 +33,8 @@
 from vllm.tokenizers.mistral import MistralTokenizer
 from vllm.utils.serial_utils import tensor2base64
 
-from ..models.registry import HF_EXAMPLE_MODELS
-from ..utils import VLLM_PATH
+from ...models.registry import HF_EXAMPLE_MODELS
+from ...utils import VLLM_PATH
```

Comment:
I suggest using `from tests.utils` here.

Reply:
I've updated all the imports from utils in `tests/entrypoints`